tests/mockWidgets.js

```diff
@@ -46,7 +46,7 @@ const MockRenderer = new Knowledge.Class({
     },
 });
 
-const MockEknWebview = new Knowledge.Class({
+var MockEknWebview = new Knowledge.Class({
```

Comment:
out of curiosity, why changing this and not the other Mock classes?

Reply:
I didn't see a warning for the others, but I'll go ahead and change it anyway.